code/datums/quirks/neutral.dm

```diff
@@ -39,14 +39,16 @@
 /datum/quirk/foreigner/add()
 	var/mob/living/carbon/human/human_holder = quirk_holder
 	human_holder.add_blocked_language(/datum/language/common)
-	if(ishumanbasic(human_holder))
-		human_holder.grant_language(/datum/language/uncommon)
+	//SKYRAT EDIT: Fixes most people not getting Galactic Uncommon.
+	//if(ishumanbasic(human_holder))
+	human_holder.grant_language(/datum/language/uncommon)
 
 /datum/quirk/foreigner/remove()
 	var/mob/living/carbon/human/human_holder = quirk_holder
 	human_holder.remove_blocked_language(/datum/language/common)
-	if(ishumanbasic(human_holder))
-		human_holder.remove_language(/datum/language/uncommon)
+	//SKYRAT EDIT: Fixes most people not getting Galactic Uncommon.
+	//if(ishumanbasic(human_holder))
+	human_holder.remove_language(/datum/language/uncommon)
```

Comment:
For this, I would recommend checking that it's not already in their background that they know the language, otherwise they will lose it in case it ever gets taken away from them.

Reply:
I flipped through backgrounds and nothing seems to give access to Galactic Uncommon.

Reply:
Yuh, but in case there's ever a chance someone has it one way or another, surely there's a way to check where the language came from, no?

Reply:
Fair